### PR TITLE
readme: add unmaintained notice

### PR DESCRIPTION
**Pull request by cgonyeo**

### CODE REVIEW

**Comment by sanmai-NL** [comment]
Could you provide links to the alternative tools please? Also, could you merge the open PRs? I'm a bit let down I spent time on it.

**Comment by dgonyeo** [comment]
I'm so sorry you spent time on this only for me to throw up an unmaintained notice, it's something I should've done sooner. I just want to be clear with people about the status of this project before they invest too much in it.

I certainly think acbuild is still a useful tool and there's space for it in the ecosystem, I just don't personally have the time anymore to work on it. I'd be happy to work with someone else to give them access to maintain the project if someone is willing to step up.

I added a thing to the PR pointing people to [umoci](https://github.com/openSUSE/umoci), which is the closest thing to acbuild that I'm aware of.

**Comment by vbatts** [comment]
https://github.com/projectatomic/buildah is now similar to acbuild

**Comment by dgonyeo** [comment]
Oh cool, I wasn't aware of that project yet. Added a link to it.

**Comment by cdrage** [comment]
Even though that `build` is unmaintained, why does `rkt` still recommend to use acbuild for building containers? Seems kind of odd that rkt links to an un-maintained tool... [1]

Bit disappointed too since it's nice to use non-Docker tools to build non-Docker containers.

1 - https://coreos.com/rkt/docs/latest/trying-out-rkt.html

ping @dgonyeo @jonboulle

**Comment by lucab** [comment]
@cdrage that cleanup is part of https://github.com/rkt/rkt/issues/3745.

File: README.md [hunk 1]
```diff
@@ -1,3 +1,19 @@
+## _This project is currently unmaintained_
+
+acbuild was originally created to be the tool used to build AppC images. Due to
+the introduction of the [Open Container
+Initiative](https://www.opencontainers.org/), development on AppC was
+[officially suspended](https://github.com/appc/spec#-disclaimer-) in November,
+2016. While acbuild has the ability to also produce OCI images, it is not the
+only tool capable of doing so. In its current state, acbuild is not currently
+maintained. If you wish to become a maintainer of acbuild, please contact Derek
+Gonyeo at derek.gonyeo@coreos.com.
```
Comment (lucab) [inline README.md]: I would suggest to replace this "send me an email" with "feel free to start contributing and ask for direct commit access via the issue/PR tracker".

Comment (dgonyeo) [inline README.md]: done

**Review by lucab — APPROVED** [review]
LGTM

**Pull request merged**